src/featureFlags/index.tsx

```diff
@@ -38,6 +43,10 @@ function useFeatureFlagsContext(): FeatureFlagsContextType {
   }
 }
 
+function useFeatureFlagsConfig(): ConfigResult {
+  return useConfig('feature_flags')
```

Comment:
so we you should write a generic `useDynamicConfig` getter similar to `useBaseFlag` that accepts the config name, and the config names should also be an enum type similar to `FeatureFlag` 

so something like:

```
export enum DynamicConfig {
  quickRouteChains = 'quick_route_chains'
}

export function useDynamicConfig(config: DynamicConfig, defaultValue = '') {
    return useConfig(config)
}
```

something like this